lib/twinklyhaha_web/controllers/oc2_controller.ex

```diff
@@ -7,37 +7,34 @@ defmodule TwinklyhahaWeb.OC2Controller do
   @off "off"
 
   def command(conn, params) do
-    Logger.debug("oc2_controller command #{inspect(params)}")
-    ## check top level components of command json
-    tops = Map.keys(params)
-
-    cond do
-      ## is action missing?
-      "action" not in tops ->
-        ## :unprocessable_entity - 422
-        Logger.debug("command no action #{inspect(params)}")
-        Logger.debug("command no action #{inspect(conn)}")
-        send_resp(conn, :unprocessable_entity, "Oops! no action?")
-
-      ## is target missing?
-      "target" not in tops ->
-        Logger.debug("command no target #{inspect(params)}")
+    command =
+      params
+      |> Jason.encode!()
+      |> Openc2.Oc2.Command.new()
+      |> Openc2.Oc2.Command.do_cmd()
+    case command.target do
```

Comment:
I would prefer we rely on the `%Openc2.Oc2.Command{error?: true, error_msg: "error_msg"}` error field  instead of checking if the target is `nil`

Reply:
What do you mean by the target can be passed but not among the required targets? If it's not among the required targets, then it will not be nil, right? It will be handled in the case where the target is not nil